### PR TITLE
Be explicit about the required plugin versions

### PR DESCRIPTION
**Pull request by ankon**
This fixes maven warnings at build time.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -182,7 +182,9 @@
       </plugin>
 
       <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
```
Comment (cgruber) [inline pom.xml]: This is redundant, as maven defaults to org.apache.maven.plugins.

File: pom.xml [hunk 2]
```diff
@@ -206,6 +208,12 @@
           </execution>
         </executions>
       </plugin>
+
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
```
Comment (cgruber) [inline pom.xml]: ditto.

**Comment by cgruber** [comment]
Thanks for this.  A few minor nits, and I"ll merge it.

**Comment by ankon** [comment]
I agree that the `groupId` additions could be considered "redundant", however not having them might get users with a) additional `pluginGroups` in `settings.xml` and b) actually existing plugins in those groups that overlap those from `org.apache.maven.plugins` into trouble. 

https://maven.apache.org/guides/introduction/introduction-to-plugin-prefix-mapping.html says that the search in `org.apache.maven.plugins` is only done when the plugin could not be resolved against any of the group ids given in the plugin groups configuration. Being explicit here seems to be the defensive thing to do, even though the issue is probably extremely unlikely.

Regardless: I'm fine with updating the PR, should I rebase + squash or just add the fixup commit on top of the current state?

**Comment by ronshapiro** [comment]
@cgruber did you see @ankon's comment? It sounds reasonable to me to be explicit.

**Comment by cgruber** [comment]
I did not.  I'll take care of it.

**Pull request closed without merging**